packages/cli/src/databases/utils/transformers.ts

```diff
@@ -3,8 +3,9 @@ import { ValueTransformer } from 'typeorm';
 import config from '@/config';
 
 export const idStringifier = {
-	from: (value: number): string | number => (typeof value === 'number' ? value.toString() : value),
-	to: (value: string): number | string => (typeof value === 'string' ? Number(value) : value),
+	from: (value: number): string => value?.toString(),
```

Comment:
Why the `?` operator?

Reply:
because typeorm passes in `null` sometimes. i tried debugging, and didn't get anywhere.

Reply:
Reply here: https://github.com/n8n-io/n8n/pull/5041#discussion_r1058994011